spec/package-deps-spec.coffee

```diff
@@ -88,3 +88,13 @@ describe 'Package-Deps', ->
           expect(atom.packages.activatePackage.callCount).toBe(3)
           expect(view.show.callCount).toBe(1)
           expect(view.advance.callCount).toBe(2)
+
+  describe 'helpers', ->
+
+    describe 'guessName', ->
+      Helpers = require('../lib/helpers')
```

Comment:
Asserting a path that includes your username is hostile to others contributing (the test will never pass for them). Use the `fixtures` directory and `temp`, and construct your target directory and files in a tempdir. Then assert against those. This will also allow you to have a single test instead of a UNIX test and a Windows test.